src/App.vue

```diff
@@ -2,10 +2,8 @@
   <div class="header-block">.</div>
   <nav class="navbar header" role="navigation" aria-label="main navigation">
     <div class="navbar-brand">
-      <router-link to="/" class="home-link">
-        TGI Pages - {{ version }}
```

Comment:
Oopsie, so that was what lead to the page resetting to the home page when you clicked into the search bar